app/webpack/shared/components/text_editor.jsx

```diff
@@ -54,9 +54,7 @@ class TextEditor extends React.Component {
     let textareaOnChange;
     if ( maxLength ) {
       textareaOnChange = e => {
-        if ( e.target.value.length > showCharsRemainingAt ) {
```

Comment:
This is just going to remove the `showCharsRemainingAt` functionality and always show the indicator. We need to keep that functionality while also allowing select all -> delete to remove the indicator.